cmd/ui/main.go

```diff
@@ -40,12 +40,14 @@ const (
 	defThingsAuthURL     = "localhost:8181"
 	defAuthURL           = "localhost:8189"
 	defThingsAuthTimeout = "1s"
+	defAuthToken         = "123"
 
 	envLogLevel          = "MF_GUI_LOG_LEVEL"
 	envClientTLS         = "MF_GUI_CLIENT_TLS"
 	envCACerts           = "MF_GUI_CA_CERTS"
 	envPort              = "MF_GUI_PORT"
 	envRedirectURL       = "MF_GUI_REDIRECT_URL"
+	envAuthToken         = "MF_GUI_AUTH_TOKEN"
```

Comment:
I do not see the need for this, UI will not be used in a single-user mode. You are using it just for development purposes.